airflow/utils/dag_processing.py

```diff
@@ -315,7 +318,8 @@ def list_py_file_paths(directory, safe_mode=True,
                 with open(ignore_file, 'r') as f:
                     # If we have new patterns create a copy so we don't change
                     # the previous list (which would affect other subdirs)
-                    patterns += [re.compile(p) for p in f.read().split('\n') if p]
+                    lines = [COMMENT_PATTERN.sub("", line) for line in f.read().split("\n")]
```

Comment:
LGTM. One small nit: the variable name `lines` doesn't really show the intention IMO. How about renaming it to e.g. "`lines_no_comments`"?